framer.go

```diff
@@ -16,11 +16,20 @@ const (
 	maxControlFrames = 16 << 10
 )
 
+// This is the largest possible size of a stream-related control frame
+// (which is the RESET_STREAM frame).
+const minStreamControlFrameSize = 25
```

Comment:
Should this be `maxStreamControlFrameSize` ?

Reply:
Yes indeed :)

Reply:
Done.